app/1.0/docs/devguide/data-system.md

```diff
@@ -326,16 +326,16 @@ Related tasks:
 Polymer implements the mediator pattern, where a host element manages data flow between itself and
 its local DOM nodes.
 
-When two elements are connected with a data binding, data changes can flow *downward*, that is, from
-host to target, *upward*, from target to host, or both ways.
+When two elements are connected with a data binding, data changes can flow **downward** — from
+host to target — **, upward** — from target to host — , or both.
```

Comment:
Don't combine both the dash and comma to set off a clause. Choose one or the other.

"or both" => both what? There are several possible referrents in this sentence, including "two elements" and "data changes" ... "both ways" is an extra word, but I think it removes ambiguity here.